server/index.js

```diff
@@ -1,18 +1,38 @@
 const express = require('express');
 const cors = require('cors')
+const petModel = require('./db/index');
+const mongoose = require('mongoose');
 
 const app = express();
 const port = 8000;
 
 app.use(cors())
 app.use(express.json())
 
+mongoose.connect('mongodb://localhost:27017/pet-cards', () => console.log('Connected to DB'))
```

Comment:
Please move this line in a function under dbService that Wraps mongo. and use it as a middlewear . The index.js should not be aware which db we are using. 

it should be something like app.use(dbConnect())

Reply:
Done